frontend/src/pages/Landing/footer.module.css

```diff
@@ -0,0 +1,20 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  height: 400px;
```

Comment:
If you're using fixed sizes it could create responsiveness issues unless you define specific breakpoints.